tools/gmhazard_scripts/db_creation/empirical_db/calc_distances_flt.py

```diff
@@ -1,4 +1,6 @@
 import argparse
+import time
+
 import numpy as np
 import os
```

Comment:
while there was a change in line 2~3, might as well we move `import os` to the top 👍